### ISSUE TITLE
Incompatibility with lcobucci/jwt:^4.0

### ISSUE THREAD

**Issue by costinel-draculet** [lexik/LexikJWTAuthenticationBundle#823]
The `https://github.com/lcobucci/jwt/blob/4.1.x/src/Signer/Ecdsa.php` class now has a `SignatureConverter` constructor parameter which is not sent for EC algorithms when initiating the signer instance in `getSignerForAlgorithm`

```
\Lexik\Bundle\JWTAuthenticationBundle\Services\JWSProvider\LcobucciJWSProvider
...
private function getSignerForAlgorithm($signatureAlgorithm)
    {
        $signerMap = [
            'HS256' => Signer\Hmac\Sha256::class,
            'HS384' => Signer\Hmac\Sha384::class,
            'HS512' => Signer\Hmac\Sha512::class,
            'RS256' => Signer\Rsa\Sha256::class,
            'RS384' => Signer\Rsa\Sha384::class,
            'RS512' => Signer\Rsa\Sha512::class,
            'EC256' => Signer\Ecdsa\Sha256::class,
            'EC384' => Signer\Ecdsa\Sha384::class,
            'EC512' => Signer\Ecdsa\Sha512::class,
        ];

        if (!isset($signerMap[$signatureAlgorithm])) {
            throw new \InvalidArgumentException(
                sprintf('The algorithm "%s" is not supported by %s', $signatureAlgorithm, __CLASS__)
            );
        }

        $signerClass = $signerMap[$signatureAlgorithm];

        return new $signerClass();
    }
```

This will cause error:

```
In Ecdsa.php line 15:

Too few arguments to function Lcobucci\JWT\Signer\Ecdsa::__construct(), 0 passed in /vendor/lexik/jwt-authentication-bundle/Services/JWSProvider/LcobucciJWSProvider.php on line 217 and exactly 1 expected
```

Possible solution:
`return new $signerClass(new Signer\Ecdsa\MultibyteStringConverter());` for signers that are instance of `\Lcobucci\JWT\Signer\Ecdsa`

**Comment by chalasr** [comment]
Thank you for the report, I'm going to release a fix for this in the next couple of days.

**Comment by chalasr** [comment]
Fixed in v2.10.6
